src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -207,6 +207,37 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     // of the extensions in the URLs. This allows loading assemblies with arbitrary filenames.
     assembliesBeingLoaded.forEach(r => addResourceAsAssembly(r, changeExtension(r.name, '.dll')));
     pdbsBeingLoaded.forEach(r => addResourceAsAssembly(r, r.name));
+
+    // Wire-up callbacks for satellite assemblies. Blazor will call these as part of the application
+    // startup sequence to load satellite assemblies for the application's culture.
+    window['Blazor']._internal.getSatelliteAssemblies = (dotnetArray: Pointer) : Pointer =>  {
+      const culturesToLoad = BINDING.mono_array_to_js_array<string>(dotnetArray);
+      const satelliteResources = resourceLoader.bootConfig.resources.satelliteResources;
+
+      if (satelliteResources) {
+        const resourcePromises = Promise.all(culturesToLoad
+            .filter(culture => Object.keys(satelliteResources).indexOf(culture) != -1)
+            .map(culture => resourceLoader.loadResources(satelliteResources[culture], fileName => `_framework/_bin/${fileName}`))
+            .reduce((previous, next) => previous.concat(next), new Array<LoadingResource>())
```

Comment:
```suggestion
            .flat()
```

(if I'm following correctly)